programs/ssl/ssl_client2.c

```diff
@@ -538,6 +540,42 @@ static void my_debug( void *ctx, int level,
     fflush(  (FILE *) ctx  );
 }
 
+
+mbedtls_time_t dummy_constant_time( mbedtls_time_t* time )
+{
+    (void) time;
+    return 0x5af2a056;
+}
+
+int dummy_random( void *p_rng, unsigned char *output, size_t output_len )
+{
+    int ret;
+    size_t i;
+
+    //use mbedtls_ctr_drbg_random to find bugs in it
+    ret = mbedtls_ctr_drbg_random( p_rng, output, output_len );
+    for ( i = 0; i < output_len; i++ )
+    {
+        //replace result with pseudo random
+        output[i] = (unsigned char) rand();
+    }
+    return( ret );
+}
+
+int dummy_entropy( void *data, unsigned char *output, size_t len )
+{
+    size_t i;
+    (void) data;
+
+    //ret = mbedtls_entropy_func( data, output, len );
```

Comment:
Should we call this function to find bugs in it, as was done with `mbedtls_ctr_drbg_random()`?

Reply:
I think so.
The counterpart is decreased performance...